src/GUI.py

```diff
@@ -0,0 +1,42 @@
+import tkinter as tk
+from tkinter import ttk, messagebox
+from main import generate_password
+
+def on_generate_password():
+    # Generate a password
+    password = generate_password()
+    # Update the result_label with the generated password
+    result_label.config(text=f"Generated Password: {password}")
+
+# Apply a theme
+style = ttk.Style()
+style.theme_use('clam')  # or 'alt', 'default', 'classic', 'vista', 'xpnative'
+
+# Customize button and label styles
+style.configure('TButton', font=('Arial', 10), background='#E1E1E1')
+style.configure('TLabel', font=('Arial', 10), background='#F5F5F5', foreground='#333333')
+
+root = tk.Tk()
+root.title("Abo-Leeh Password Generator")
+root.configure(bg='#F5F5F5')  # Set background color
+
+# Set a fixed window size
+root.geometry('400x200')
```

Comment:
I guess it is better to make 40x200 a global variable

Reply:
done